lib/node_modules/@stdlib/stats/incr/wstdev/docs/types/index.d.ts

```diff
@@ -0,0 +1,68 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 4.1
+
+/// <reference types="@stdlib/types"/>
+
+/**
+* If provided both arguments, returns an updated weighted sample standard deviation; otherwise, returns the current weighted sample standard deviation.
+*
+* ## Notes
+*
+* -   If provided `NaN` or a value which, when used in computations, results in `NaN`, the accumulated value is `NaN` for all future invocations.
+*
+* @param x - value
+* @param {NonNegativeNumber} [w] - weight
+* @throws {TypeError} second argument must be a nonnegative number
+* @returns weighted sample standard deviation
+*/
+type accumulator = ( x?: number, w?: number ) => number | null;
+
+/**
+* Returns an accumulator function which incrementally computes a weighted sample standard deviation.
```

Comment:
Looks like this is correct, based on the implementation, where a known mean parameter is supported. This and the tests need to be updated.